src/components/widgets/edge-widget.tsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import { SimpleServiceWidget } from './simple-service-widget';
 
-const EdgeWidget: React.FunctionComponent = () => {
+const ImagesWidget: React.FunctionComponent = () => {
```

Comment:
since you added a new widget in a separate file, I don't think we need to keep this one any longer. 

you will also need to add the new widget to the dashboard.cy.ts cypress E2E test.

Reply:
Thanks @aferd 👍 
I removed `Edge`
Also I updated the dashboard template
 https://github.com/RedHatInsights/chrome-service-backend/pull/749